docs/run-tests-locally.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[run-tests-locally]]
+== Run Tests Locally
+
+To run tests locally you can make use of the docker images also used when running the whole testsuite with Jenkins. 
+Running the full testsuite first does some linting and then runs the actual tests with different versions of python and different webframeworks. For a full overview of the test matrix and supported versions have a look at the [Jenkins configuration](Jenkinsfile). 
+
+[[linter]]
+=== Linter
+We run three kinds of linting `isort`, `flake8`, and `docs`. You can trigger each single linter locally by running:
```

Comment:
`docs` isn't really a linter IMO. It just tests if the docs can be generated without errors. Not sure how to write it differently, though